core/src/main/java/org/mule/runtime/core/api/scheduler/SchedulerConfig.java

```diff
@@ -16,6 +19,24 @@
 public class SchedulerConfig {
 
   /**
+   * Different values on how to handle the scenario where a task is dispatched to a busy {@link Scheduler}.
+   * <p>
+   * A {@link Scheduler} is considered busy when all of its threads are busy and it cannot accept a new task for execution.
+   */
+  public enum DispatchingRejectionPolicy {
+    /**
+     * The actual rejection policy will depend on the type of scheduler the thread is. For cpu-bound threads it will be
```

Comment:
Given pool types are already well-defined in SchedulerService there is no reason not to call them out specifically here by name to make things even clearer.